src/main/java/com/stripe/model/Money.java

```diff
@@ -1,8 +1,9 @@
 package com.stripe.model;
 
-public class Money {
+public class Money extends StripeObject {
 	Long amount;
 	String currency;
+  SourceTypes sourceTypes;
```

Comment:
So it's definitely a little unusual, but the Java code here uses tabs instead of spaces. Maybe just do a global find and replace on all the files modified in your diff.